client/me/purchases/cancel-purchase/button.jsx

```diff
@@ -254,7 +254,7 @@ class CancelPurchaseButton extends Component {
 
 		let text, onClick;
 
-		if ( isRefundable( purchase ) ) {
+		if ( isRefundable( purchase ) && purchase.refundAmount > 0 ) {
```

Comment:
There are other `isRefundable()` calls in this file -- should they be changed too, for consistency?  (I think they might not get called in the case where `purchase.refundAmount` is 0 anyway, but if they ever did it looks like they might do the wrong thing.)

Reply:
Replaced those calls with `hasAmountAvailableToRefund` :)